server/views/serviceProviderReferrals/interventionProgress.njk

```diff
@@ -56,7 +56,64 @@
   </p>
 
   {% if presenter.hasSessions %}
-    {{ govukTable(sessionTableArgs(govukTag)) }}
+  <table class="govuk-table" id="sessionAppointmentsTable">
```

Comment:
adding data-cy should let you select this table in the cypress tests without changing the selector
`<table data-cy="session-table" class="govuk-table" id="sessionAppointmentsTable">`